CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Bugfix: Fixed a bug that caused all badge highlights to use the same color. (#3132, #3134)
 - Dev: Renamed CMake's build option `USE_SYSTEM_QT5KEYCHAIN` to `USE_SYSTEM_QTKEYCHAIN`. (#3103)
 - Dev: Add benchmarks that can be compiled with the `BUILD_BENCHMARKS` CMake flag. Off by default. (#3038)
+- Bugfix: Use `flatpak-spawn` to call streamlink when running as a Flatapk. Fixes streamlink functionality. (#3178)
```

Comment:
I feel like this changelog entry isn't very user-friendly. Maybe change to something along the lines of:
```suggestion
- Bugfix: Allow starting Streamlink from Chatterino when running as a Flatpak. (#3178)
```

(cc: @Felanbird @ALazyMeme for grammar checks :smile:)

Reply:
It's in the dev section anyway I was just gonna suggest changing it to `Dev:`, but this works too

Reply:
> It's in the dev section

Good catch, didn't even notice that. I still feel like listing this as a "Bugfix" entry adds more value to changelog readers rather than it just being a "Dev" entry